topo_processor/metadata/metadata_validators/metadata_validator_tiff.py

```diff
@@ -23,31 +23,32 @@ def is_applicable(self, item: Item) -> bool:
         return False
 
     def validate_metadata(self, item: Item) -> None:
-        geospatial_type = item.properties["linz:photo_type"]
+
+        geospatial_type = item.properties["linz:geospatial_type"]
         for asset in item.assets:
             if not is_tiff(asset.source_path):
                 continue
             eo_bands = asset.properties["eo:bands"]
             with rasterio.open(asset.source_path) as tiff:
                 # black and white
                 if ColorInterp.gray in tiff.colorinterp and len(tiff.colorinterp) == 1:
-                    # check linz:photo_type matches colorinterp
-                    if geospatial_type != "B&W":
+                    # check linz:geospatial_type matches colorinterp
+                    if geospatial_type != "black and white image":
```

Comment:
I would create a an `Enum` for those types. Easier to have them centralized somewhere for code maintenance.

Reply:
Just a note that this will be refactored later.